src/libraries/System.Net.Http/src/System/Net/Http/BrowserHttpHandler/BrowserHttpHandler.cs

```diff
@@ -95,14 +116,27 @@ public bool PreAuthenticate
 
         public bool AllowAutoRedirect
         {
-            get => throw new PlatformNotSupportedException();
-            set => throw new PlatformNotSupportedException();
+            get => _allowAutoRedirect;
+            set
+            {
+                CheckDisposedOrStarted();
+                _allowAutoRedirect = value;
+            }
         }
 
         public int MaxAutomaticRedirections
         {
-            get => throw new PlatformNotSupportedException();
-            set => throw new PlatformNotSupportedException();
+            get => _maxAutomaticRedirections;
```

Comment:
Is it used anywhere?
I see only `AllowAutoRedirect` in `SendAsync`.

Reply:
Is that not part of the SupportsRedirectConfiguration.  If not we can move it back to PlatformNotSupported since it is not used.  Meaning fetch does not use it as it does not support it.

Reply:
I see. We could just change the exception type and keep it throwing. To let the user know that this settings doesn't have any effect.

Although, I'm not sure what's better here. I'll leave it up to you to decide.

Reply:
Leaving it at Platform Not Supported Exception since it is not supported by platform fetch api.  It may be confusing to see this set and fetch not respecting the value.